docs/_templates/components_requirements.tmpl

```diff
@@ -3,17 +3,27 @@
 Inmanta core and extensions versions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+The orchestrator is constituted of several components (python packages and npm packages).
+The python packages are of two types: one core package and several extension packages.
 
 {% if data['component_requirements']%}
+
+The following table shows the range of compatible versions for Inmanta python components.
```

Comment:
I think we should make it clear that the production version is the starting point and from that we define version constraints on the versions of the components.
```suggestion
The following table shows the ranges of the Inmanta Python components, this product version is compatible with.
```